convert-gpt4all-to-ggml.py

```diff
@@ -32,6 +33,7 @@ def write_header(f_out, header):
 
     if magic != 0x67676d6c:
```

Comment:
Can you make the magic a constant too?

Reply:
We need old magic constants anyway to detect older models.